scripts/deploy-federation.sh

```diff
@@ -83,6 +83,13 @@ fi
 shift
 JOIN_CLUSTERS="${*}"
 
+# Use DOCKER_PUSH=false ./scripts/deploy-federation.sh <image> to skip docker
+# push on container image when not using latest image.
+DOCKER_PUSH=${DOCKER_PUSH:-docker push ${IMAGE_NAME}}
+if [[ "${DOCKER_PUSH}" == false ]]; then
```

Comment:
(Action not necessary): Consider using a different flag for the user intent.
Say a user thinks this to be a flag and provides `DOCKER_PUSH=true ./script/deploy-federation.sh`

Reply:
+1 to using a flag separate from the command.  Also, consider using the pattern of `FLAG=y|[empty]` and `[[ -z "${FLAG}" ]]` instead of requiring a specific value.